nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -420,18 +424,24 @@ def _impl(inputs, attr, params):
             ignores=['index_type', 'T'])(new_inputs, attr)
     return _impl
 
+def _split():
+    def _impl(inputs, attr, params):
+        pop_node = inputs.pop(0)
+        axis = params[pop_node.list_output_names()[0]].asnumpy()[0]
+        return AttrCvt(op_name="split",
+                       ignores=['num_split'],
```

Comment:
Suggest to use transforms instead of ignores & extras.

Reply:
Have you reached consensus the changes here is good?